Mage/src/main/java/mage/abilities/keyword/ExtortAbility.java

```diff
@@ -32,7 +26,10 @@
 public class ExtortAbility extends TriggeredAbilityImpl {
 
     public ExtortAbility() {
-        super(Zone.BATTLEFIELD, new ExtortEffect(), false);
+        super(Zone.BATTLEFIELD, new DoIfCostPaid(
+                new LoseLifeOpponentsYouGainLifeLostEffect(1),
+                new ManaCostsImpl<>("{W/B}"),
+                "Pay {WB} to Extort?"));
```

Comment:
{W/B} instead? Or am I missing something about {WB}?

Reply:
Oh huh interesting, I was going off of the comments which frequently use the pattern without a slash, but yeah, it should definitely be using one looking at actual code elsewhere.

Reply:
On testing it appears that both options provide the correct hybrid mana symbol. For consistency I'll use the one with the slash though.